testsuite/regressiontests/ticket-689.sli

```diff
@@ -42,8 +42,8 @@ M_ERROR setverbosity
   /total_num_virtual_procs 1
 >> SetKernelStatus
 
-/iaf_psc_alpha Create /n1 Set
-/iaf_psc_alpha Create /n2 Set
+/iaf_psc_alpha_hom Create /n1 Set
```

Comment:
Should this remain the original? Should the test for the "hom" version be added rather than replacing the original?